pywbemtools/pywbemcli/_context_obj.py

```diff
@@ -281,7 +281,7 @@ def execute_cmd(self, cmd):
             self.spinner_start()
         if self.pdb:
             import pdb  # pylint: disable=import-outside-toplevel
-            pdb.set_trace()
+            pdb.set_trace()  # pylint: disable=no-member
```

Comment:
This Pylint disable is no longer needed, since it now works in your new virtual env.

Reply:
DONE all of these now removed.  I guess I must have missed a rebase after the update to the other pr

Reply:
I later figured out that the pylint issue is caused when pdb++ is installed and filed an issue.